test/es-module/test-esm-wasm.mjs

```diff
@@ -90,4 +90,160 @@ describe('ESM: WASM modules', { concurrency: !process.env.TEST_PARALLEL }, () =>
     match(stderr, /ExperimentalWarning/);
     match(stderr, /WebAssembly/);
   });
+
+  it('should support static source phase imports', async () => {
+    const { code, stderr, stdout } = await spawnPromisified(execPath, [
+      '--no-warnings',
+      '--experimental-wasm-modules',
+      '--input-type=module',
+      '--eval',
+      [
+        'import { strictEqual } from "node:assert";',
+        `import * as wasmExports from ${JSON.stringify(fixtures.fileURL('es-modules/wasm-source-phase.js'))};`,
```

Comment:
Drive-by; shouldn't this say `import source * as wasmExports`?

Reply:
This test case is a re-export of a source phase import.